### PR TITLE
Pundeep9/bugfix/endianness fix to adcs get full config

### PR DESCRIPTION
**Pull request by pundeep9**
The endianness of the ADCS_get_full_config function was changed. Changes were also made to adcs_binary_test to test the endianness fix. Any changes outside of adcs_handler and adcs_binary_testing should be discarded. Unrelating changes were also made to the adcs_binary testing file which will be useful for future testing.

### CODE REVIEW

**Comment by Taylorr82** [comment]
> The endianness of the ADCS_get_full_config function was changed. Changes were also made to adcs_binary_test to test the endianness fix. Any changes outside of adcs_handler and adcs_binary_testing should be discarded. Unrelating changes were also made to the adcs_binary testing file which will be useful for future testing.

If the changes should be discarded why are they here

**Comment by pundeep9** [comment]
> > The endianness of the ADCS_get_full_config function was changed. Changes were also made to adcs_binary_test to test the endianness fix. Any changes outside of adcs_handler and adcs_binary_testing should be discarded. Unrelating changes were also made to the adcs_binary testing file which will be useful for future testing.
> 
> If the changes should be discarded why are they here

There should be none outside of adcs files. There were places where we had to make changes to other parts of the code (mostly UHF stuff) to get everything to work. I made sure none of that would make it onto the git but in case some squeezed in, I want to make sure that comments to resolve those are not made. I think there shouldn't be any though.

File: main/version.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#ifndef VERSION_H
```
Comment (Taylorr82) [inline main/version.h:1]: Version.h should not be checked in

File: .git-hooks/hooks/pre-commit [hunk 2]
```diff
@@ -1,36 +0,0 @@
-#!/bin/bash
```
Comment (Taylorr82) [inline .git-hooks/hooks/pre-commit:1]: Why is this file deleted?

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

File: .git-hooks/hooks/pre-commit (hunk 2 above)
Comment (pundeep9) [reply .git-hooks/hooks/pre-commit:1]: Because my computer does not let me push changes if I have that file. It's specific to my device so I just get rid of it, hence I said only look at adcs files.

Comment (Taylorr82) [reply .git-hooks/hooks/pre-commit:1]: I can't just look at adcs files. When this is merged it brings ALL changes, not just the ones you wanted me to see. Also, your computer does not let you push on purpose, you need to install clang-format

File: ex2_hal/adcs/equipment_handler/src/adcs_handler.c [hunk 3]
```diff
@@ -3100,74 +3105,211 @@ ADCS_returnState ADCS_get_full_config(adcs_config *config) {
...
+
+    config->cubesense.cam1_sense.exposure_t = ((telemetry[47] << 8) | telemetry[46]); // WRONG >>> FIXED
+
     config->cubesense.cam1_sense.boresight_x = ((telemetry[49] << 8) | telemetry[48]) * coef;
     config->cubesense.cam1_sense.boresight_y = ((telemetry[51] << 8) | telemetry[50]) * coef;
     get_xyz(&config->cubesense.cam2_sense.mounting_angle, &telemetry[52], 0.01);
     config->cubesense.cam2_sense.detect_th = telemetry[58];
     config->cubesense.cam2_sense.auto_adjust = telemetry[59] & 1;
-    memcpy(&config->cubesense.cam2_sense.exposure_t, &telemetry[60], 2);
+    config->cubesense.cam2_sense.exposure_t = ((telemetry[61] << 8) | telemetry[60]); // WRONG >>> FIXED
```
Comment (Taylorr82) [inline ex2_hal/adcs/equipment_handler/src/adcs_handler.c:3134]: // WRONG >>> FIXED

This isn't a helpful comment. We know it's wrong, that's why this commit exists

File: ex2_hal/adcs/equipment_handler/src/adcs_handler.c [hunk 4]
```diff
@@ -3100,74 +3105,211 @@ ADCS_returnState ADCS_get_full_config(adcs_config *config) {
...
+        }
+    }
+    memcpy(&config->rwheel.proportional_gain, &temp_rwheel[0], 4);
+    memcpy(&config->rwheel.derivative_gain, &temp_rwheel[1], 4);
+    memcpy(&config->rwheel.bias_moment, &temp_rwheel[2], 4);
+
     config->rwheel.sun_point_facet = telemetry[309] & 0x7F; // 7 bits
     config->rwheel.auto_transit = telemetry[309] & 0x80;    // 8th bit
 
+    // This whole function needs to be seriously reworked
```
Comment (Taylorr82) [inline ex2_hal/adcs/equipment_handler/src/adcs_handler.c:3251]: Add TODO so this is searchable

File: ex2_hal/adcs/equipment_handler/test/adcs_binary_test.c [hunk 5]
```diff
@@ -3419,7 +3420,7 @@ void commandsTest_unix_configID(void)
     printf("When = %u \n", when);
     printf("Period = %u \n", period);
 
-    uint32_t time = 1651165879;
+    uint32_t time = 0;
```
Comment (Taylorr82) [inline ex2_hal/adcs/equipment_handler/test/adcs_binary_test.c:3423]: Why did you erase a valid time?

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

**Comment by Taylorr82** [comment]
Looks mostly okay. I'm going to rebase it and fix up the rest
